docs/data/sdks/java/index.md

```diff
@@ -46,7 +46,22 @@ Amplitude client = Amplitude.getInstance("YOUR_INSTANCE_NAME");
 client.init("YOUR_API_KEY");
 ```
 
-### Configure batching behavior
+### Configuration
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `serverUrl` | The server url events upload to. | `https://api2.amplitude.com/2/httpapi` |
+    | `minIdLength` | The minimum length for user id or device id. | `5` |
```

Comment:
is this supported in Java now?

Reply:
Yes, they setup way is a little bit different. 
```
    amplitude.setOptions(new Options()
            .addHeader("Custom Header", "value")
            .setMinIdLength(5));
```
I will update it to options.minIdLength.

Reply:
[Pr](https://github.com/amplitude/Amplitude-Java/pull/63) for reference.